webui/src/pages/repositories/repository/pulls/pullDetails.jsx

```diff
@@ -45,6 +47,14 @@ const PullDetailsContent = ({repo, pull}) => {
     let [loading, setLoading] = useState(false);
     let [error, setError] = useState(null);
 
+    const {state: {results: diffResults, loading: diffLoading, error: diffError}} = useContext(DiffContext);
+    const isEmptyDiff = (!diffLoading && !diffError && !!diffResults && diffResults.length === 0);
+    // this is pretty hacky, but there seem to be no other way to detect this specific error
+    const isBranchNotFoundError =
+        diffError?.message?.startsWith("get commit by ref") &&
+        diffError?.message?.endsWith("not found");
```

Comment:
This is only a partial solution. This does not work when deleting destination branch
Also shows the error double
![image](https://github.com/user-attachments/assets/0c7421d7-e7c2-4774-bdb5-659deb04da8d)

Reply:
Also let's be more verbose about which branch is not found

Reply:
And this is what happens when we try to merge:
![image](https://github.com/user-attachments/assets/e2d81a1d-25e7-4fb2-9763-f7d510342c43)